test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -201,18 +201,25 @@ public static void ClearProfilerEnvironmentVariables()
         {
             var processName = processPath;
             string profilerEnabled = _requiresProfiling ? "1" : "0";
+            string profilerPath;
```

Comment:
nit: looks like you can assign this here and remove the ones below
```suggestion
            string profilerPath = GetProfilerPath();
```

Reply:
Good catch, I'll fix that.

Reply:
Fixed in 7a2a2ce4d640c750eca578a28ef7c32cde1eadb4 .